collection_requirements.rst

```diff
@@ -337,10 +337,8 @@ CI Testing
   * Reviewers can block acceptance of a new collection if they don't agree with the ignores.txt entries.
 
 * You MUST run CI against each of the "major versions" (2.10, 2.11, 2.12, etc) of ``ansible-base``/``ansible-core`` that the collection supports. (Usually the ``HEAD`` of the stable-xxx branches.)
-
 * All CI tests MUST run against every pull request and SHOULD pass before merge.
-* All CI tests MUST pass for the commit that releases the collection.
- 
+* At least sanity tests MUST run against a commit that releases the collection (if the collection has integration/unit tests, they SHOULD run too); if they don't pass, the collection will NOT be released.
```

Comment:
+1

would it make sense to even specify which sanity tests to run? e.g. sanity tests of latest 2 stable branches?